assets/js/components/TourTooltips.test.js

```diff
@@ -134,18 +135,12 @@ describe( 'TourTooltips', () => {
 		);
 	} );
 
-	it( 'should switch to next step when next button is clicked', async () => {
+	it( 'should switch to next step when next button is clicked', () => {
 		const { getByRole } = renderTourTooltipsWithMockUI( registry );
 
 		fireEvent.click( getByRole( 'button', { name: /next/i } ) );
 
 		getByRole( 'heading', { name: /title for step 2/i } );
-
-		await act( waitForDefaultTimeouts );
-
-		// There is a minor bug in the `TourTooltips` component that results in a "React state update on an unmounted component"
-		// warning. This will be addressed in a followup issue, see https://github.com/google/site-kit-wp/issues/6345
-		expect( console ).toHaveErrored();
```

Comment:
This assertion was introduced due to an error that surfaced apparently due to the dependency changes introduced in #6203 (see 539319dad84dd305382f22595a5df2e676ee2506). It appears the dependency changes in this PR have negated the need for this.